src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  if (days > 2) {
```

Comment:
The condition `if (days > 2)` is redundant because the subsequent conditions `if (days >= 7)` and `if (days >= 3)` already cover all cases. You can remove this outer condition to simplify the code.